### DISCUSSION TITLE
local transport in mountainous/hilly terrain

### DISCUSSION THREAD

**Discussion by inku0120** [uataq/stilt#126] [Users]
Dear Sir, I am trying to visualize the effect of local transport to the receptor site. 
The observation station at our receptor site is located at very high altitude (hilly area) such a way that boundary layer lies very close to surface or sometimes lies below the observation station.

My doubt is in such a case, can I use STILT to understand the influence of local transport to the receptor site, if yes what parameters i need to be taken care of i run_stilt.r file. 
What value of PBL i should use?? Will STILT be able to provide some significant results?   

OR i should run WRF and then use the output for STILT?? 

Thank you in advance.

**Comment by tartanrunner25** [comment]
STILT could be utilized for this application. However, whether the STILT simulations could accurately replicate such a situation would be dependent on whether this situation could be captured by the host 3-D model. My guess is something like WRF would be needed to capture high-resolution mountain/hill circulations. You could then run STILT off of WRF output files. See: 

Lin, J. C., D. V. Mallia, D. Wu, and B. Stephens: How can mountaintop CO2 observations be used
to constrain regional carbon fluxes? Atmos. Chem. Phys., 17, 5561–5581, 2017.
